src/outlineView.ts

```diff
@@ -111,16 +118,25 @@ export class OutlineView {
     }
 
     // add new cursorOn attribue
-    const cursorPoint = newBufferPosition.row
+    const cursorPoint = cursor.getBufferRow()
     this.focusedElms = this.pointToElementsMap.get(cursorPoint)
 
     if (this.focusedElms !== undefined) {
+      this.focusedElms[0].scrollIntoView({
```

Comment:
You sure it always contains elements? Maybe
```suggestion
      this.focusedElms[0]?.scrollIntoView({
```
?

Reply:
This is impossible. It is either `undefined` or a full array.
https://github.com/atom-community/atom-ide-outline/blob/3e0c6d04dfdadd60d4bd1c32d7966ac694ade220/src/outlineView.ts#L345-L351